pepper-apis/src/main/java/org/broadinstitute/ddp/route/ReceiveDsmNotificationRoute.java

```diff
@@ -97,6 +97,7 @@ public Object handle(Request request, Response response, DsmNotificationPayload
                     user.getId(),
                     user.getId(),
                     userGuid,
+                    userGuid,
```

Comment:
There are situations where we do not have an operator, such as services executed by background jobs or API endpoints that are called by other system components rather than an operator user. In these cases, it's not clear to me who/what is the operator, and whether it makes sense to allow study configuration to use "operator PEX" in these cases. So I think it will be preferable to NOT assume operator == participant and instead pass in `null` for operator guid.

The `ReceiveDsmNotificationRoute` is one where we do not have an operator.

Reply:
Done